### ISSUE TITLE
JSON missing from lib directory in 1.0.0

### ISSUE THREAD

**Issue by lencioni** [evcohen/eslint-plugin-jsx-a11y#40]
I'm working on upgrading something to use 1.0.0, and am running into this error:

```
Error: Failed to load plugin jsx-a11y: Cannot find module '../util/attributes/ARIA'
```

I believe this is because the build step doesn't bring the JSON files over to the lib directory from src.

**Comment by evcohen** [comment]
ah fixing now. you're right. just manually doing a `cp` command for that now. hmm why can't babel handle that?

**Comment by evcohen** [comment]
Should be fixed in v1.0.1

**Comment by lencioni** [comment]
Thanks for the fast fix! FYI https://github.com/airbnb/javascript/pull/838
